web_autoflip_demo/autoflip_demo/autoflip_worker.ts

```diff
@@ -0,0 +1,93 @@
+importScripts('visual_design_service_web_assembly_api.js');
+const ctx: any = self;
+
+var wait: number = 0;
```

Comment:
Don't use var, only use let or const. Use let in place of var.

Reply:
fixed.